packages/MarkdownEditor-Tests.package/MarkdownCommentBlockTests.class/instance/testClosingTagFollowedByTextCloses.st

```diff
@@ -0,0 +1,7 @@
+tests - extendability
+testClosingTagFollowedByTextCloses
+
+	| commentBlock |
+	commentBlock := MarkdownCommentBlock newFrom: '<!-- MiniMetro', Character cr.
+	commentBlock append: 'is great --> more Text'.
```

Comment:
```smalltalk
	commentBlock := MarkdownCommentBlock newFrom: '<!-- Comment with', Character cr.
	commentBlock append: 'multiple lines --> Text after end tag'.
```
Applies to other `isClosed` and `commentEnd` tests

Reply:
Meh. I think it's pretty clear like it is...